src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsService.kt

```diff
@@ -1090,7 +1090,6 @@ private fun PunishmentsAdditionalInformation.toTelemetryMap(): MutableMap<String
 )
 
 internal fun ReportedAdjudicationResponse.toNomisAdjudication() = CreateAdjudicationRequest(
-  adjudicationNumber = reportedAdjudication.chargeNumber.toNomisAdjudicationNumber(),
```

Comment:
no longer supply adjudicationNumber or the companion offenceId, these are now generated by nomis api and returned